var/spack/repos/builtin/packages/openblas/package.py

```diff
@@ -96,6 +96,12 @@ class Openblas(MakefilePackage):
     # Fix ICE in LLVM 9.0.0 https://github.com/xianyi/OpenBLAS/pull/2329
     # Patch as in https://github.com/xianyi/OpenBLAS/pull/2597
     patch('openblas_appleclang11.patch', when='@0.3.8:0.3.9 %apple-clang@11.0.3')
+    # There was an error in Reference-LAPACK that is triggeret by Xcode12
+    # fixed upstream by https://github.com/xianyi/OpenBLAS/pull/2808 and
+    # should be included in post 0.3.10 versions. Application to earlier
+    # versions was not tested.
+    # See also https://github.com/xianyi/OpenBLAS/issues/2870
+    patch('openblas-0.3.10-appleclang12.patch', when='@:0.3.10 %apple-clang@12.0.0:')
```

Comment:
```suggestion
    patch('openblas-0.3.10-appleclang12.patch', when='@0.3.8:0.3.10 %apple-clang@12:')
```

Just tried with `spack patch openblas@...` and the patch is only applicable @ `0.3.[8-10]`.

Reply:
done